hail/python/test/hail/methods/test_impex.py

```diff
@@ -1150,9 +1150,7 @@ def test_contig_recoding_defaults(self):
             reference_genome='GRCh38',
         )
 
-        rg_random = hl.ReferenceGenome(
-            "random", ['1', '23', '24', '25', '26'], {'1': 10, '23': 10, '24': 10, '25': 10, '26': 10}
-        )
+        hl.ReferenceGenome("random", ['1', '23', '24', '25', '26'], {'1': 10, '23': 10, '24': 10, '25': 10, '26': 10})
```

Comment:
This is necessary, but I quite dislike that we made the `ReferenceGenome` constructor side effect. :/